README.md

```diff
@@ -38,8 +38,9 @@ This package adds functionalities to the Eloquent model and Query builder for Mo
         - [Basic Usage](#basic-usage-2)
         - [Available operations](#available-operations)
     - [Transactions](#transactions)
-    - [Schema](#schema)
         - [Basic Usage](#basic-usage-3)
+    - [Schema](#schema)
+        - [Basic Usage](#basic-usage-4)
```

Comment:
Why is the basic-usage-4?

Reply:
basic-usage-3 points to the previous section.
I think someone added another basic usage and forgot to update this section